digdag-core/src/main/java/io/digdag/core/schedule/ScheduleExecutor.java

```diff
@@ -199,8 +199,14 @@ private void runSchedule(ScheduleControl lockedSched)
 
             Config scheduleConfig = SchedulerManager.getScheduleConfig(def);
             boolean skipOnOvertime = scheduleConfig.get("skip_on_overtime", boolean.class, false);
+            boolean disableBackfill = scheduleConfig.get("disable_backfill", boolean.class, false);
 
-            if (!activeAttempts.isEmpty() && skipOnOvertime) {
+            // task should run at scheduled time within one second delay if disable_backfill is true.
+            if (disableBackfill && now.isAfter(sched.getNextRunTime().plusSeconds(1))) {
```

Comment:
I think the best and safest delay depends on use cases and situations. For instance, if Digdag is handling a lot of tasks, it's likely Digdag tries to start handling a task whose next run time is already a few second ago even in normal situation.

How about making the delay configurable not boolean?

Reply:
Skipping tasks that have recent `StoredSchedule#getCreatedAt` might also work.

Reply:
OK, I'll change the keyword from `disable_backfill` to `backfill_limit`.
e.g. forbid backfill execution if it's over 30 seconds from scheduled time.
backfill_limit: 30